ginga/misc/plugins/ChangeHistory.py

```diff
@@ -256,3 +256,10 @@ def start(self):
 
     def __str__(self):
         return 'changehistory'
+
+
+# Replace module docstring with config doc for auto insert by Sphinx.
+# In the future, if we need the real docstring, we can append instead of
+# overwrite.
+from ginga.util.toolbox import generate_cfg_example
+__doc__ = generate_cfg_example('plugin_ChangeHistory', package='ginga')
```

Comment:
It might take a little time to update all the plugins--plugins without the incantation just show the documentation without the config file?